src/component/MissionState/MissionState.tsx

```diff
@@ -1,17 +1,48 @@
-import React, {useEffect, useState} from "react"
+import CompoundedSpace from "antd/lib/space";
+import {useContext, useEffect, useState} from "react"
+import { FlightsContext } from "../../Providers/FlightsProvider/FlightsProvider";
 import "./MissionState.css"
 
-function MissionState() {
-    const [date, updateDate] = useState(0)
+const months = ["stycznia", "lutego", "marca", "kwietnia", "maja", "czerwca", "lipca", "sierpnia", "września", "października", "listopada", "grudnia"] as const;
+
+type TMissionState = "WAITING" | "LIVE" | "ENDED"
```

Comment:
I see that later strings from this type are used in code. Maybe replace this with enum? Then you may use this as a type and retrive value from it. This can be useful if one of these values ​​needs to be changed. Wdyt?